CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing to ShadowNode
+
+## Overview
+ShadowNode is one of the Node.js runtimes for edge device, support for most [Node.js API](https://nodejs.org/en/docs/) such as `Event`, `Module` and `N-API`. We plan to support the most of packages in NPM for Node.js in the future. If you are interested in joining it, we will welcome it very much.
+
+## Getting and Building
+Firstly, you can only build it in Linux and macOS now.
+1. Install the following prerequisites, as necessary:
+  - The tools needed to build ShadowNode from scratch:
+    - Standard C development headers
+    - Git 1.9+
+    - CMake 3.1+
+    - Node.js and NPM
+    - DBus(Ubuntu is Builted-in)
+
+Note that at least 4GB of RAM is required to build from source and run tests.
+
+2. Get the ShadowNode code:
```

Comment:
I don't think this step is necessary in this document since it's a guidance to build ShadowNode. In this stage we'd assume developers already got hands on ShadowNode source codes.